style.css

```diff
@@ -0,0 +1,52 @@
+.first-section {
+  border: 4px solid red;
+  font-size: 1.2rem;
+  margin-bottom: 20px;
+}
+
+.section-heading {
```

Comment:
We don't need this extra class. Kindly remove this and use more accurate solution for this.
CC: @nb-1231